### DISCUSSION TITLE
Thumbnails for Loras

### DISCUSSION THREAD

**Discussion by GodwinF** [lllyasviel/Fooocus#3197] [Ideas]
Hi, I have a huge list of loras and sometimes its difficult to know what the lora does. 
I think it would be really helpful to have some method of associating a thumbnail for the lora and display it in the interface. I know styles have thumbnails and they display when you hover over them, but i would like to see them all in the interface without hovering over them one by one. something like a gallery where it shows all the thumbnails of styles and loras.

**Comment by bsdesigner** [comment]
Yes, it would be nice if there was a preview of checkpoint and Lore miniatures. But it would be much cooler if when adding this or that Lore, automatically add trigger words to the prompt.

**Comment by mashb1t** [comment]
See https://github.com/lllyasviel/Fooocus/issues/2029 and https://github.com/lllyasviel/Fooocus/pull/2211, but i personally don't think this is the best way to implement it. Feel free to shoot your ideas!

**Reply by bsdesigner** [reply]
> See #2029 and #2211, but i personally don't think this is the best way to implement it. Feel free to shoot your ideas!

Personally, I don't really need the preview, as I use Fooocus in the Stability Matrix package manager, and they are fine with the preview. And by the way, since you're already here, I wanted to inform you that I've repeatedly asked Stability Matrix developers to include your fork "Fooocus - mashb1t's 1-Up Edition" in their platform, but unfortunately they never responded to these requests, Although their package includes such outdated forks as RuinedFooocus, Fooocus-ControlNet and Fooocus-MRE

https://github.com/LykosAI/StabilityMatrix/discussions/675

![зображення_2024-06-28_225709265](https://github.com/lllyasviel/Fooocus/assets/36730003/007d4c3f-920f-4d05-81e2-1cde5db01348)

**Reply by ILikeToasters** [reply]
Just to let you know that they are aware of the requests to add his fork but they are more active in their Discord and it has been discussed and I think there is a good chance of it being added soon - "not at this time, though we've had a lot of requests for that fork so might make it in one of the next versions"
